utils/providers.py

```diff
@@ -307,14 +309,19 @@ def _setup_providers(prov_class, validate, check_existing):
 
     """
 
+    prov_class_map = {
```

Comment:
There is another way to do this, because those are imported at the top you can do this,
```python
from cfme.common.provider import BaseProvider
```
and then the ```prov_class_map``` IS ```BaseProvider.type_mapping```